pkg/apis/extensions/types.go

```diff
@@ -87,6 +88,18 @@ type CPUTargetUtilization struct {
 	TargetPercentage int `json:"targetPercentage"`
 }
 
+// Alpha-level support for Custom Metrics in HPA (as annotations).
+type CustomMetricUtilization struct {
```

Comment:
Sorry, I must have missed the comment. I will rename the Struct, but probably to CustomMetricTarget as there will be another CM-related struct for status.